nextflow_schema.json

```diff
@@ -305,14 +305,14 @@
                 "custom_config_version": {
                     "type": "string",
                     "description": "Git commit id for Institutional configs.",
-                    "default": "master",
+                    "default": "main",
                     "hidden": true,
                     "fa_icon": "fas fa-users-cog"
                 },
                 "custom_config_base": {
                     "type": "string",
                     "description": "Base directory for Institutional configs.",
-                    "default": "https://raw.githubusercontent.com/nf-core/configs/master",
+                    "default": "https://raw.githubusercontent.com/nf-core/configs/main",
```

Comment:
This should remain as `master`.

Reply:
As above [3d6e79a](https://github.com/phac-nml/staramrnf/pull/20/commits/3d6e79a22a089010389cd526b300080787384089)